skxray/core/tests/test_roi.py

```diff
@@ -278,49 +276,35 @@ def test_static_test_sets():
         int_array[int_array == 0] = i*100
         images2.append(int_array)
 
-    samples = np.array((np.asarray(images1), np.asarray(images2)))
+    samples = {'sample1': np.asarray(images1), 'sample2': np.asarray(images2)}
 
     roi_data = np.array(([2, 30, 12, 15], [40, 20, 15, 10]), dtype=np.int64)
 
     label_array = roi.rectangles(roi_data, shape=(50, 50))
 
-    # test mean_intensity function
-    average_intensity, index = roi.mean_intensity(np.asarray(images1),
-                                                  label_array)
-    # test mean_intensity_sets function
-    average_int_sets, index_list = roi.mean_intensity_sets(samples,
-                                                           label_array)
-
-    assert_array_equal((list(average_int_sets)[0][:, 0]),
-                       [float(x) for x in range(0, 1000, 100)])
-    assert_array_equal((list(average_int_sets)[1][:, 0]),
-                       [float(x) for x in range(0, 20, 1)])
-
-    assert_array_equal((list(average_int_sets)[0][:, 1]),
-                       [float(x) for x in range(0, 10, 1)])
-    assert_array_equal((list(average_int_sets)[1][:, 1]),
-                       [float(x) for x in range(0, 2000, 100)])
-
-    # test combine_mean_intensity function
-    combine_mean_int = roi.combine_mean_intensity(average_int_sets,
-                                                  index_list)
-
-    roi_data2 = np.array(([2, 30, 12, 15], [40, 20, 15, 10],
-                          [20, 2, 4, 5]), dtype=np.int64)
-
-    label_array2 = roi.rectangles(roi_data2, shape=(50, 50))
-
-    average_int2, index2 = roi.mean_intensity(np.asarray(images1),
-                                              label_array2)
-    index_list2 = [index_list, index2]
-
-    average_int_sets.append(average_int2)
-
-    # raise ValueError when there is different labels in different image sets
-    #  when trying to combine the values
-    assert_raises(ValueError,
-                  lambda: roi.combine_mean_intensity(average_int_sets,
-                                                     index_list2))
+    # get the mean intensities of image sets given as a dictionary
+    roi_data = {}
+    for k, v in sorted(samples.items()):
+        intensity, index_list = roi.mean_intensity(v, label_array)
+        roi_data[k] = intensity
+    average_int_sets = pd.DataFrame(roi_data, index_list)
```

Comment:
Can you rework this test to not use pandas?  I am not sure that it is a good idea to pick up a pandas dependency in the test suite because of this minor use.